### ISSUE TITLE
Windows 10 detected by JVM if running on Windows 11

### ISSUE THREAD

**Issue by rks2003** [adoptium/adoptium-support#577]

**Comment by jerboaa** [comment]
This is https://bugs.openjdk.org/browse/JDK-8274840 which would need a backport to 8u.

**Comment by karianna** [comment]
Odd - https://bugs.openjdk.org/browse/JDK-8279280 seems to think the backport is done or is that OracleJDK only in the tagging system?

**Comment by jerboaa** [comment]
> Odd - https://bugs.openjdk.org/browse/JDK-8279280 seems to think the backport is done or is that OracleJDK only in the tagging system?

Yes, Oracle JDK versions are `8uXXX`. OpenJDK versions are `openjdk8uXXX`.

**Comment by jerboaa** [comment]
Should be fixed once `8u362` gets released (January 2023)

**Comment by karianna** [comment]
@rks2003 Can you update to 1.8.0_362 and let us know?

**Comment by rks2003** [comment]
Yes, the Windows version is now detected correctly on Windows 11, great job! In the mean time I observered the same problem on Windows Server 2022 but currently do not have such an installation to verify. Should the detection of this Windows version be corrected, too?

**Comment by karianna** [comment]
Yes that seems to be fixed in internal tests
